crates/proto/src/lib.rs

```diff
@@ -4,5 +4,12 @@ use serde::{Deserialize, Serialize};
 pub struct Rustacean {
     pub name: String,
     pub image: String,
+    pub bio:String,
     pub github_url: String,
+    pub email:String,
+    pub geographical_location: String,
```

Comment:
For this I think it would be nice to have normalized values.

Listing a full set of locations could be memory expensive, using an API as well, so for now I think we could have an approximate location and limit to County and Continent.

These could be filled as pairs, Continent could be "required" but Country optional.

```suggestion
    pub location: Location,
```

and then

```rust
struct Location {
  continent: Continent,
  country: Country,
}
```

The downside is that we would have to write these by hand (or generate these using a from a CSV), but the good part is that we would do it once (and perhaps when some of this data changes due to political reasons).

The benefit here is that normalized data will allow us to perform better filtering later, as well as taking advantage of its precision for things like adding emojis for instance or locating in a map, or perhaps reusing some ISO bits to perform timezone matching for a range of TZs in a Country space.